### PR TITLE
added multiply function

### PR DESCRIPTION
**Pull request by alpyurtseven**

### CODE REVIEW

File: createRepoTest.js [hunk 1]
```diff
@@ -10,4 +10,10 @@ function subtract (){
     Array.from(arguments).slice(1).reduce((total, value)=>{
         return total -=value;
     },arguments[0]);
+}
+
+function subtract (){
```
Comment (digitsontr) [inline createRepoTest.js:15]: Function name should be changed to multiply

**Review by digitsontr — CHANGES_REQUESTED** [review]
Function name should be changed to multiply

File: createRepoTest.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ function subtract (){
     },arguments[0]);
 }
 
-function subtract (){
+function multiply (){
     return Array.from(arguments).reduce((total, value)=>{
         return total *=value;
```
Comment (Fy1717) [inline createRepoTest.js:17]: test review comment

**Review by Fy1717 — CHANGES_REQUESTED** [review]

File: createRepoTest.js (hunk 2 above)
Comment (alpyurtseven) [reply createRepoTest.js:17]: Resoved !

**Review by digitsontr — APPROVED** [review]
Thanks

**Pull request merged**